recipes/aiida-orca/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "aiida-orca" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/aiida-orca-{{ version }}.tar.gz
+  sha256: 6cda668213376e44a29aacb197f2915d8fcc9e3394372b7029a3030432c7f8d1
+
+build:
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+    - reentry
+  run:
+    - aiida-core >=1.0.0,<2.0.0
+    - cclib
+    # pandas has broken builds: https://github.com/conda-forge/pandas-feedstock/issues/137
+    - pandas !=1.4.0,!=1.4.1,!=1.4.2
```

Comment:
> where did you get it from?

its a "deeper" dependency that was flagged on the `pip check` failure 

> how serious is this metadata error?

and yeh I imagine it is certainly serious, since you end up with a version of pytz imcompatible with pandas (according to their requirements), and so anything trying to run pandas may fail.

> I would recommend going without these and fixing with a repodata patch as suggested in the issue you link

Yeh ughh, it would be nice if the pandas team did it, given that it is such a problematic issue that they have introduced 😅 I doubt it is just an issue with this package
but maybe, if I have time to work all that out